lib/mpl_toolkits/mplot3d/art3d.py

```diff
@@ -28,22 +28,40 @@
 
 
 def norm_angle(a):
-    """Return angle between -180 and +180"""
+    """Return the given angle normalized to -180 < *a* <= 180 degrees."""
     a = (a + 360) % 360
     if a > 180:
         a = a - 360
     return a
 
 
 def norm_text_angle(a):
-    """Return angle between -90 and +90"""
+    """Return the given angle normalized to -90 < *a* <= 90 degrees."""
     a = (a + 180) % 180
     if a > 90:
         a = a - 180
     return a
 
 
 def get_dir_vector(zdir):
+    """
+    Return a direction vector.
+
+    Parameters
+    ----------
+    zdir : {'x', 'y', 'z', None, 3-tuple}
+        The direction. Possible values are:
+        - 'x': equivalent to (1, 0, 0)
+        - 'y': euqivalent to (0, 1, 0)
+        - 'z': equivalent to (0, 0, 1)
+        - *None*: euqivalent to (0, 0, 0)
+        - an iterable (x, y, z) is returned unchanged.
+
+    Returns
+    -------
+    np.array((x, y, z))
```

Comment:
slight quibble, if a tuple was passed in, then a tuple comes back. So array-like, instead?